### ISSUE TITLE
[문승태] Today I Learned — DS — 10일차 — Join에 대하여

### ISSUE THREAD

**Issue by seungtaemoon** [codestates/ds-blog#81]
안녕하세요 여러분

제가 오늘 정리한 join에 따른 merge와 concat의 사용 의의에 대해 정리한 블로그를 공유합니다.

링크: [Today I Learned — DS — 10일차 — Join에 대하여](https://medium.com/@conanmoon/today-i-learned-ds-10%EC%9D%BC%EC%B0%A8-join%EC%97%90-%EB%8C%80%ED%95%98%EC%97%AC-5f9b574a0102)

한번 읽어주시고, 언제든지 피드백 환영입니다.

오늘 다들 수고하셨습니다~!

**Comment by jhk0530** [comment]
concat 에서도 join을 쓸 수 있는 줄은 승태님 덕분에 처음 알았습니다.

결론과 이후 부분에서도 요약이랑 추가 내용들을 잘 적어 주셔서, 제가 다 뿌듯해지네요. 

이번 주도 고생 많이 하셨습니다. 다음주에 뵈어요

**Comment by seungtaemoon** [comment]
> concat 에서도 join을 쓸 수 있는 줄은 승태님 덕분에 처음 알았습니다.
> 
> 결론과 이후 부분에서도 요약이랑 추가 내용들을 잘 적어 주셔서, 제가 다 뿌듯해지네요.
> 
> 이번 주도 고생 많이 하셨습니다. 다음주에 뵈어요

네 ㅋㅋ 다 인터넷 덕분입니다. 판다스의 함수는 파라미터들이 생각보다 많더라구요, 거기서 조금만 조정해주면 많은 기능들을 사용할 수 있는 것 같아요!

처음 글을 쓸 떼 이 내용들이 Join의 개념들과 관련이 깊어보여서 같이 엮어서 글을 쓰면 좋을 것 같다는 생각을 했어요~

칭찬 감사드립니다 진환님, 앞으로 더 좋은 글을 쓰기 위해 노력할께요~!
